mr/src/main/java/org/elasticsearch/hadoop/rest/query/TermQueryBuilder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.hadoop.rest.query;
+
+import org.elasticsearch.hadoop.serialization.Generator;
+
+public class TermQueryBuilder extends QueryBuilder {
+    private String field;
+    private String term;
+
+    public TermQueryBuilder field(String field) {
+        this.field = field;
+        return this;
+    }
+
+    public TermQueryBuilder term(String term) {
+        this.term = term;
+        return this;
+    }
```

Comment:
Is there ever a point in time where it makes sense for either of these values to be left uninitialized? I think that it should throw an illegal state exception if it's called without them being set. Better yet, maybe it makes sense to force the user to acknowledge the properties as missing by making both fields part of the constructor/static factory method?